deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -33,6 +33,10 @@ Parameters:
   AppConfigPath:
     Type: String
     Description: The path to look for the application config from
+  MarbleProcessingKeyPath:
+    Type: String
+    Description: The ssm path to look for the marble data processing config from
+    Default: '/all/marble-data-processing/test'
```

Comment:
Presumably, we will have a script that will override this default to pass '/all/marble-data-processing/prod' for production.

Reply:
I am going to change this so that it rely's on the production pathway and have the local deploy send the test values in.